File: Day339/MinimumChangesToMakeAlternateBinaryStrings.go
func minOperations(s string) int {
    a,b := 0, 0
    for i:=0; i<len(s); i++ {
        if i%2 == 0 {
            if s[i] == '0' {
                a++
            }
            if s[i] == '1' {
                b++
            }
        } else {
            if s[i] == '1' {
                a++
            }
            if s[i] == '0' {
                b++
            }
        }
    }
    return int(math.Min(float64(a), float64(b)))
}

File: Day339/MinimumChangesToMakeAlternateBinaryStrings.java
class Solution {
    public int minOperations(String s) {
        int a = 0, b = 0;
        for (int i=0; i<s.length(); i++) {
            if (i%2 == 0) {
                if (s.charAt(i) == '0') {
                    a++;
                }
                if (s.charAt(i) == '1') {
                    b++;
                }
            } else {
                if (s.charAt(i) == '1') {
                    a++;
                }
                if (s.charAt(i)== '0') {
                    b++;
                }
            }
        }
        return Math.min(a, b);
    }
}